server/gql/models/studentSubjects/customUpdateResolver.js

```diff
@@ -0,0 +1,27 @@
+import { updateStudent, updateSubject, updateStudentSubject } from '@server/daos/studentSubjects';
+import { transformSQLError } from '@server/utils';
+
+export const customUpdateResolver = async (model, args, context) => {
+  try {
+    if (args?.studentName) {
+      const studentArgs = {
+        id: args.studentId,
+        name: args.studentName
+      };
+      delete args.studentName;
+      await updateStudent(studentArgs);
+    }
+    if (args?.subjectName) {
+      const subjectArgs = {
+        id: args.subjectId,
+        name: args.subjectName
+      };
+      delete args.subjectName;
+      await updateSubject(subjectArgs);
+    }
+    const res = await updateStudentSubject(args);
```

Comment:
over you can just return this 
```
return updateStudentSubject(args);
```

```
const res = await ...
return res;
```
is redundant

Reply:
Okay.

Reply:
Yes.

Reply:
Yes.